app/elements/candidates/invite-candidate-input.html

```diff
@@ -54,28 +45,44 @@
 				challenges: {
 					type: Array
 				},
-				challenge: {
-					type: Object
-				},
 				candidate: {
 					type: Object,
 					reflectToAttribute: true,
 					notify: true
-				},
-				singleChallenge: {
-					type: Boolean,
-					computed: 'isSingleChallenge(challenge)'
 				}
 			},
-			isSingleChallenge: function(challenge){
-				return challenge !== undefined;
-			},
-			_setChallenge: function(){
+			observers: [
+				'_afterProperties(challenges, candidate)'
+			],
+			_setChallenge: function () {
 				this.candidate.Challenge = this.$.challenges.options[this.$.challenges.selectedIndex].value;
 			},
-			_removeItem: function(){
+			_removeItem: function () {
 				this.candidate.Removed = true;
 				this.remove();
+			},
+			_afterProperties: function () {
+				var option = document.createElement('option');
+				if (this.challenges.length === 1) {
+					option.value = this.challenges[0].Key;
+					option.innerHTML = this.challenges[0].Name;
```

Comment:
No, because i'd rather it be consistent. Select in both cases, just that in one there is only one already selected option(when you open this modal from the challenge-detail page).